java/java.sourceui/src/org/netbeans/api/java/source/ui/snippet/MarkupTag.java

```diff
@@ -0,0 +1,28 @@
+package org.netbeans.api.java.source.ui.snippet;
```

Comment:
Note that there are no license headers - this can't pass through Apache RAT. Similarly many other files which are missing the headers (some test files are excluded - please run RAT and check: `ant rat`).

Reply:
Addressed.